packages/components/src/components/hds/form/radio/group.ts

```diff
@@ -0,0 +1,32 @@
+import templateOnlyComponent from '@ember/component/template-only';
+import type { HdsFormFieldsetSignature } from '../fieldset';
+import type { ComponentLike } from '@glint/template';
+import type { HdsFormLegendSignature } from '../legend';
+import type { HdsFormHelperTextSignature } from '../helper-text';
+import type { HdsFormRadioFieldSignature } from './field';
+import type { HdsFormErrorSignature } from '../error';
+
+interface HdsFormRadioGroupSignature {
+  Args: {
+    layout?: HdsFormFieldsetSignature['Args']['layout'];
+    name?: string;
+    isRequired?: boolean;
+    isOptional?: boolean;
+  };
```

Comment:
would it make sense to do something like this? (here and in other places)
```
  Args: HdsFormFieldsetSignature['Args'] & {
    name?: string;
  };
```
or better to be explicit and see the list of arguments in this context? in that case, should we do something like this? (here and in other places)
```
  Args: {
    layout?: HdsFormFieldsetSignature['Args']['layout'];
    isRequired?: HdsFormFieldsetSignature['Args']['isRequired'];
    isOptional?: HdsFormFieldsetSignature['Args']['isOptional'];
    name?: string;
  };
```

Reply:
yes, that's a good call! I will update to extend the Field/Fieldset signatures

Reply:
updated in 1b8c5a4 and f5dbe5b